pip_audit/_cli.py

```diff
@@ -134,6 +134,32 @@ def __str__(self) -> str:
         return self.value
 
 
+@enum.unique
+class VulnerabilityAliasChoice(str, enum.Enum):
+    """
+    Whether or not vulnerability aliases should be added to the `pip-audit` output.
+    """
+
+    On = "on"
+    Off = "off"
+    Auto = "auto"
+
+    def to_bool(self, format_: OutputFormatChoice) -> bool:
+        if format_ is OutputFormatChoice.Json:  # Note: Aliases are always rendered for JSON.
+            return True
```

Comment:
Per discussion above: let's remove this, for consistent with the other format-modifying flags.

```suggestion
```